spec/scripts/aamva_cert_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.describe 'AAMVA cert script' do
+  before do
+    allow(IdentityConfig.store).to receive(:proofer_mock_fallback).and_return(false)
+    allow(IdentityConfig.store).to receive(:aamva_private_key).
+      and_return(Base64.strict_encode64(AamvaFixtures.aamva_private_key.to_der))
+    allow(IdentityConfig.store).to receive(:aamva_public_key).
+      and_return(Base64.strict_encode64(AamvaFixtures.aamva_public_key.to_der))
+  end
+
+  # This script can be run against the DLDV test environment (with the correct URLs)
+  subject(:run_script) do
```

Comment:
another idea would be to put this in `scripts/` dir something, so the script itself and the test are separate? but this seemed easy enough since the expectation is that we'd copy-paste most of this in the console anyways

Reply:
Seems like it would be quicker to find in the scripts directory while in the middle of an incident. And your comment talks about running it in the DLDV test env. This also works in prod, right?

Is this the Jonny Proofer data, or is that somewhere else?

Reply:
This was focused on having a DLDV test cert script, rather than a script we could use to check connectivity in prod, so now I think there are two modes that we could use:
1. DLDV auth test
2. one-off "jonny proofs" test to check prod connectivity

Reply:
made a big refactor in f8276397dfddeb90f281d98b99b8f0d75769d293, now we have two scripts:

- `bin/aamva-test-cert --auth-url=URL --verification-url=URL` which does the cert testing
- `bin/aamva-test-connectivity` which just sends Jonny Proofs